changelog.txt

```diff
@@ -5,6 +5,7 @@
 * Fix - Prevent payment gateways that complete Change Payment requests asynchronously from blocking future attempts to update payment methods for all subscriptions.
 * Fix - Resolved an issue that could lead to undefined variable when attempting to send a customer notification email with no order.
 * Fix - Prevents the customer's postal code being used as their billing city in the current session, following a change to payment details.
+* Fix - Safeguards added to the Subscriptions Totals template used in the My Account area, to guard against fatal errors that could arise in unusual conditions.
```

Comment:
Small change needed here. This changelog needs to be moved to the next version as 7.9 was shipped last week 😄

Reply:
Sure thing! Added a new section for 8.0.0 and moved it there → https://github.com/Automattic/woocommerce-subscriptions-core/pull/759/commits/68da33640c80ec59cecd2838a9e7e04fced22e05